bears/xml/XMLBear.py

```diff
@@ -24,10 +26,16 @@ class XMLBear(LocalBear, Lint):
     gives_corrected = True
 
     def process_output(self, output, filename, file):
-        if self.stdout_output:  # only yield Result if stdout is not empty
-            return self._process_corrected(self.stdout_output, filename, file)
-        if self.stderr_output:  # pragma: no cover
+        if self.stderr_output:
+            self.use_stderr = True
+            if self.stdout_output:  # only yield Result if stdout is not empty
+                return collections.ChainMap(
+                           self._process_issues(self.stderr_output, filename),
```

Comment:
@AbdealiJK What's the issue in indentation ?
Isn't it 4 spaces PEP8 doesn't give me any error.